src/main/java/controller/LoginController.java

```diff
@@ -0,0 +1,57 @@
+package controller;
+
+import db.DataBase;
+import http.HttpQueryParams;
+import http.HttpRequest;
+import http.HttpResponse;
+import model.User;
+import session.Session;
+import session.SessionManager;
+
+import java.util.UUID;
+
+public class LoginController extends Controller {
+    private static final String INDEX_PAGE = "/index.html";
+    private static final String LOGIN_PAGE = "/user/login.html";
+    private static final String LOGIN_FAILED_PAGE = "/user/login_failed.html";
+
+    protected void doGet(HttpRequest request, HttpResponse response) {
+        String sessionId = request.getSessionId();
+
+        Session session = SessionManager.findSession(sessionId);
+
+        if (session != null) {
+            response.redirect(INDEX_PAGE);
+            return;
+        }
+        response.redirect(LOGIN_PAGE);
+    }
+
+    protected void doPost(HttpRequest request, HttpResponse response) {
+        String body = (String) request.getBody();
+        HttpQueryParams queryParams = HttpQueryParams.parseParams(body);
+        handleLogin(queryParams, response);
+    }
+
+    private void handleLogin(HttpQueryParams queryParams, HttpResponse response) {
+        String userId = queryParams.get("userId");
+        String password = queryParams.get("password");
+
+        User existUser = DataBase.findUserById(userId);
+
+        if (existUser == null || !existUser.match(userId, password)) {
+            response.redirect(LOGIN_FAILED_PAGE);
+            return;
+        }
+
+        String uuid = UUID.randomUUID().toString();
+        response.addCookie("JSESSIONID", uuid);
```

Comment:
가져온 혹은 새로 생성한 Session 객체를 그대로 addCookie의 메서드 파라미터로 전달해주면 어떨까요? 🙂

```suggestion
        response.addCookie(session);
```

`JSESSIONID`는 필요한 위치에 상수로 추출해주면 좋을 것 같아요. 🙂